src/zepben/evolve/model/cim/iec61970/base/wires/transformer_star_impedance.py

```diff
@@ -0,0 +1,35 @@
+#  Copyright 2021 Zeppelin Bend Pty Ltd
+#
+#  This Source Code Form is subject to the terms of the Mozilla Public
+#  License, v. 2.0. If a copy of the MPL was not distributed with this
+#  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+from __future__ import annotations
+
+from typing import Optional, TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from zepben.evolve.model.cim.iec61968.assetinfo.transformer_end_info import TransformerEndInfo
+
+from zepben.evolve.model.cim.iec61970.base.core.identified_object import IdentifiedObject
+
+__all__ = ["TransformerStarImpedance"]
+
+
+class TransformerStarImpedance(IdentifiedObject):
+    """Transformer star impedance (Pi-model) that accurately reflects impedance for transformers with 2 or 3 windings. For transformers with 4 or more windings,
+     TransformerMeshImpedance class shall be used.
+
+    For transmission networks use PowerTransformerEnd impedances (r, r0, x, x0, b, b0, g and g0).
+
+    Attributes -
+        r : Resistance of the transformer end.
+        r0 : Zero sequence series resistance of the transformer end.
+        x : Positive sequence series reactance of the transformer end.
+        x0 : Zero sequence series reactance of the transformer end."""
+    r: float = 0.0
```

Comment:
I do have a question here about changing these values to be ints.
These values are new but the other r, x, .... resistance, reactance, etc values in the model are floats. For example: PerLengthSequenceImpedance.

Is the plan to refactor all resistance, reactance, susceptance, impedance values to be ints??

Because making them ints in these new classes I am adding will make it inconsistent with the rest of the Model.

Reply:
yes, that would be the plan. fine with the inconsistency for now (this inconsistency already exists all over the place)

Reply:
Following the pattern in the sdk-jvm and the data model documentation, r, x, r0, x0 were defined as a float.